src/Components/Onboarding/ItemLink.tsx

```diff
@@ -51,13 +51,14 @@ export default class ItemLink extends React.Component<Props, null> {
     null
   }
 
-  onClick() {
-    this.followItem()
-  }
+  // onClick() {
+  //   // this.followItem()
+  //   this.prop
+  // }
 
   render() {
     return (
-      <Link onClick={this.onClick.bind(this)}>
+      <Link onClick={this.props.onClick.bind(this)}>
```

Comment:
Does this callback really need to be rebound to `this`? It’s usually not great if you rebind a function you get from elsewhere, as it may break the working of the function if it was already bound to the right context object.

Reply:
:+1: